app/models/job_invocation.rb

```diff
@@ -15,7 +15,35 @@ class JobInvocation < ActiveRecord::Base
 
   scoped_search :on => [:job_name], :complete_value => true
 
+  attr_accessor :start_before
+  attr_writer :start_at
+
   def to_action_input
     { :id => id, :name => job_name }
   end
+
+  def execution_type
```

Comment:
I know I've came initially with this naming, but I'm still not sure that's the best name for the value we are collecting, Anyone suggestions on better name?